webapp/src/ts/services/enketo.service.ts

```diff
@@ -427,12 +454,19 @@ export class EnketoService {
         this.getUserContact(),
       ])
       .then(() => {
-        return this.renderForm(selector, form, instanceData, editedListener, valuechangeListener);
+        const contactFormContext: ContactFormContext = {
```

Comment:
I don't think "ContactFormContext" is a good name for this variable or interface, as Enketo doesn't only handle contact forms, it also handles report forms. We should use a more generalized name, like "formContext" or "enketoFormContext".